src/components/TopicPage/Articles.jsx

```diff
@@ -32,12 +34,7 @@ function Articles({ queries: { sort_by, order } }) {
   }
 
   if (isError) {
-    return (
-      <p>
-        An error has occured and the page cannot be loaded at the moment.
-        Apologies for the inconvenience.
-      </p>
-    );
+    return <h2>{err}</h2>;
```

Comment:
Although this does show the error, it also still shows the comments form, and allows me to attempt to post a comment for a non-existent article (although this of course eventually fails).

the comments list also perpetually says 'loading comments' although this should at some point say no comments or an error once the request has unsuccessfully completed, so it may be worth having a look at your error handling for requesting comments